tools/amd_build/pyHIPIFY/hipify-python.py

```diff
@@ -471,6 +473,53 @@ def replace_math_functions(input_string):
     output_string = re.sub("std::pow\(", "::pow(", output_string)
     return output_string
 
+
```

Comment:
remove empty line?

Reply:
My IDE's linter loved to scream at me haha.

Apparently it's good style to have two lines between function definitions in PyThon. https://www.python.org/dev/peps/pep-0008/?#blank-lines